xline/src/main.rs

```diff
@@ -205,6 +201,12 @@ struct ServerArgs {
     /// Curp client retry timeout
     #[clap(long, value_parser = parse_duration, default_value = "50ms")]
     client_retry_timeout: Duration,
+    /// Storage engine
+    #[clap(long)]
+    storage_engine: String,
+    /// DB directory
+    #[clap(long, default_value = "/usr/local/xline/data-dir")]
```

Comment:
Our config file and arguments parsing are a bit confusing now. Their default values are not common. For example, there is no default value for data_dir in the configuration file but it is in the arguments parsing, and some default values ​​use default_xxx in the configuration file, but in the arguments parsing directly uses a string, they may be different

Reply:
Fixed